testdata/docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# This Dockerfile generates a build environment for generating ELFs
+# of testdata programs. Run `make build` in this directory to build it.
+FROM debian:buster
+
+RUN apt-get update && \
```

Comment:
Nit: merge with the apt get below for one less layer?

Reply:
`gnupg` is required for `apt-key`, so not sure if possible.

Reply:
Yup, doesn't work. `apt-get update` doesn't complete if the key isn't installed yet. Minimizing for fewer layers is not very useful, by the way. The goal is to introduce as little data to the image per layer as possible, so removing apt cache after an apt-get in the same `RUN` statement is ideal.